cmd/server/server.go

```diff
@@ -0,0 +1,150 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	proxy "github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/spacemeshos/post/rpc"
+	"github.com/spacemeshos/post/rpc/api"
+	"github.com/spacemeshos/post/shared"
+	"github.com/spacemeshos/smutil/log"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/peer"
+	"net"
+	"net/http"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "server",
+	Short: "start server",
+	Run: func(cmd *cobra.Command, args []string) {
+		s := NewPostServer()
+
+		logger, err := s.Initialize(cmd, args)
+		if err != nil {
+			panic(err)
```

Comment:
maybe we can do a shutdown with cleanup instead of panicking? doesn't have to be as part of this pr

Reply:
There's no need for a cleanup yet, but I changed the panic into log and quit instead.